style.css

```diff
@@ -1,8 +1,10 @@
 /* mobiles */
 
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
 @media only screen and (max-width: 700px) {
```

Comment:
## This is optional but highly advised. Remove the media query and build your project as mobile first. This would improve the accessibility of your site and it is a good practise.